### DISCUSSION TITLE
How to get incoming(join call) request prompt alert in Azure communication service in React js for web, so that we can add "Admit" and "reject" option.

### DISCUSSION THREAD

**Discussion by anupsamanta** [community/community#127989] [Programming Help]
### Select Topic Area

Bug

### Body

We have implemented video calling using Azure communication services (used this link : https://github.com/Azure-Samples/communication-services-web-calling-hero). Now we can connect each others in video call automatically. But we need to implement "Admit", "Admit All" before the establishment of the connection and "Reject" functionality . Need assistance to implement this functionality .

How we can get notified a join call request. Which function should we use and where to use.

**Comment by ramkrishnasamanta8** [comment]
I got the same issue , but not getting any solution . Any help will be highly appreciated.

**Comment by developerbaba443** [comment]
Admit and Admit all functionality unable to implement, Please suggest.
